### PR TITLE
Enable optional client authentication to support mutual TLS with CQL storage backend [full build]

### LINKED ISSUE #1324: No configuration to connect to cassandra with two-way SSL
It should be possible to connect a janusgraph client to a cassandra cluster which is set up with `require_client_auth: true` . Currently, the janusgraph configuration supports setting `storage.cassandra.ssl.truststore` parameters, but not for keystore.

This was tested with janusgraph 0.2.0 and the `cassandra` backend, but after reviewing the code it appears to be the same with janusgraph 0.3.1 and the `cql` backend.

Is there some configuration I'm missing or would this be a new feature?

### PR DESCRIPTION
**Pull request by porunov**
A plain copy paste of the PR #1900 made by @callajd with resolved history problems.
In case @callajd resolves the history problems in the PR #1900 before this PR is merged, I encourage to close this PR and merge that PR. Otherwise, I suggest to merge this PR, so that we have this feature in `0.5.0` release.

Fixes #1324
Suppresses #1900 

Signed-off-by: Oleksandr Porunov <alexandr.porunov@gmail.com>

-----

Thank you for contributing to JanusGraph!

In order to streamline the review of the contribution we ask you
to ensure the following steps have been taken:

### For all changes:
- [x] Is there an issue associated with this PR? Is it referenced in the commit message?
- [x] Does your PR body contain #xyz where xyz is the issue number you are trying to resolve?
- [x] Has your PR been rebased against the latest commit within the target branch (typically `master`)?
- [x] Is your initial contribution a single, squashed commit?

### For code changes:
- [ ] Have you written and/or updated unit tests to verify your changes?
- [ ] If adding new dependencies to the code, are these dependencies licensed in a way that is compatible for inclusion under [ASF 2.0](https://www.apache.org/legal/resolved.html#category-a)?
- [ ] If applicable, have you updated the LICENSE.txt file, including the main LICENSE.txt file in the root of this repository?
- [ ] If applicable, have you updated the NOTICE.txt file, including the main NOTICE.txt file found in the root of this repository?

### For documentation related changes:
- [ ] Have you ensured that format looks appropriate for the output in which it is rendered?
- [ ] If this PR is a documentation-only change, have you added a `[doc only]`
  tag to the first line of your commit message to avoid spending CPU cycles in
  Travis CI when no code, tests, or build configuration are modified?

### Note:
Please ensure that once the PR is submitted, you check Travis CI for build issues and submit an update to your PR as soon as possible.

### CODE REVIEW

File: janusgraph-cql/src/test/resources/docker/update_config.py [hunk 1]
```diff
@@ -16,9 +16,12 @@
 
 import os,re
 
-keystore = '/etc/ssl/test.keystore'
+keystore = '/etc/ssl/node.keystore'
 keystore_password = 'cassandra'
+truststore = '/etc/ssl/node.truststore'
+truststore_password = 'cassandra'
```
Comment (FlorianHockmann) [inline janusgraph-cql/src/test/resources/docker/update_config.py]: Codacy Issue found: [Possible hardcoded password: 'cassandra'](https://app.codacy.com/manual/JanusGraph/janusgraph/pullRequest?prid=5086038)

**Comment by FlorianHockmann** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           

Clones added
============
- janusgraph-cql/src/test/resources/docker/update_config.py  3
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/JanusGraph/janusgraph/pullRequest?prid=5086038&bid=16732802)

**Comment by FlorianHockmann** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           

Clones added
============
- janusgraph-cql/src/test/resources/docker/update_config.py  3
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/JanusGraph/janusgraph/pullRequest?prid=5086038&bid=16732802)

File: janusgraph-cql/src/test/resources/docker/update_config.py (hunk 1 above)
Comment (FlorianHockmann) [inline janusgraph-cql/src/test/resources/docker/update_config.py]: Codacy Issue found: [Possible hardcoded password: 'cassandra'](https://app.codacy.com/manual/JanusGraph/janusgraph/pullRequest?prid=5086038)

**Comment by porunov** [comment]
@farodin91 @FlorianHockmann I changed the commit author and used commit signature from past signed commits by @callajd . As @callajd already signed EasyCLA by using that signature, I think it should works well.

**Comment by farodin91** [comment]
@porunov If you use interactive rebase you can squash all commits into one and don't lose any committer. This process is not so straight forward.

**Comment by porunov** [comment]
@farodin91 As I understand we don't loose any committer right now because the squashed commit is now signed and authored by @callajd . Or do you mean something else? What do we lose right now?

**Comment by porunov** [comment]
@farodin91 @FlorianHockmann tell what you think about it. If that is OK, I will merge this PR after the tests are completed

**Comment by farodin91** [comment]
@porunov This is what i'm thinking about. Sorry didn't checkout the commit.

**Pull request merged**